frontend/src/metabase/ui/components/navigation/Tabs/Tabs.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Fragment } from "react";
```

Comment:
In figma tabs have `border-radius` in all corners:

![image](https://github.com/metabase/metabase/assets/6830683/07be3bd3-08a7-48e2-adf2-277beb16f243)

In the implementation, the corners touching the line don't have `border-radius`:
![image](https://github.com/metabase/metabase/assets/6830683/e5a32b28-05ac-46a7-b949-d3c76f5b698e)

I like the implementation more though.

Reply:
Agree. I think that might've been an oversight in Figma. I'll correct it there to match what's in this PR.